test/Object/rlexe.xml

```diff
@@ -171,7 +171,7 @@
   <test>
     <default>
       <files>Slow.js</files>
-      <tags>exclude_chk</tags>
+      <tags>exclude_chk,slow</tags>
```

Comment:
Not sure whether tags are case-sensitive or not, but we've always used uppercase `Slow` for slow tests.